packages/video_player/video_player_avfoundation/ios/Classes/messages.g.h

```diff
@@ -1,79 +1,95 @@
 // Copyright 2013 The Flutter Authors. All rights reserved.
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
-
-// Autogenerated from Pigeon (v0.1.21), do not edit directly.
+// Autogenerated from Pigeon (v1.0.16), do not edit directly.
 // See also: https://pub.dev/packages/pigeon
 #import <Foundation/Foundation.h>
 @protocol FlutterBinaryMessenger;
+@protocol FlutterMessageCodec;
 @class FlutterError;
 @class FlutterStandardTypedData;
 
 NS_ASSUME_NONNULL_BEGIN
 
 @class FLTTextureMessage;
-@class FLTCreateMessage;
 @class FLTLoopingMessage;
 @class FLTVolumeMessage;
 @class FLTPlaybackSpeedMessage;
 @class FLTPositionMessage;
+@class FLTCreateMessage;
 @class FLTMixWithOthersMessage;
 
 @interface FLTTextureMessage : NSObject
++ (instancetype)makeWithTextureId:(nullable NSNumber *)textureId;
 @property(nonatomic, strong, nullable) NSNumber *textureId;
 @end
 
-@interface FLTCreateMessage : NSObject
-@property(nonatomic, copy, nullable) NSString *asset;
-@property(nonatomic, copy, nullable) NSString *uri;
-@property(nonatomic, copy, nullable) NSString *packageName;
-@property(nonatomic, copy, nullable) NSString *formatHint;
-@property(nonatomic, strong, nullable) NSDictionary *httpHeaders;
-@end
-
 @interface FLTLoopingMessage : NSObject
++ (instancetype)makeWithTextureId:(nullable NSNumber *)textureId
```

Comment:
Ah shoot, good call.  The reason I didn't use an `init` method is because you can't use properties in there and I didn't want to generate special logic for talking to raw ivars.  "make" was chosen since it follows the create rule.  I was thinking there was some prior art for it but there isn't.  The problem is that renaming it is technically a breaking change.

Reply:
Filed https://github.com/flutter/flutter/issues/97832 to capture this.